vmngclient/api/versions_utils.py

```diff
@@ -93,20 +110,72 @@ def get_image_version(self, software_image: str) -> Union[str, None]:
         logger.error(f"Software image {image_name} is not in available images")
         return None
 
+    def _create_callback(self, encoder: MultipartEncoder):
+
+        bar = ProgressBar(expected_size=encoder._calculate_length(), filled_char="=")
+
+        def callback(monitor: MultipartEncoderMonitor):
+            bar.show(monitor.bytes_read)
+
+        return callback
+
+    def upload_image(self, image_path: str) -> int:
```

Comment:
Maybe example of upload and delete image in doc sting?

Reply:
I think, that those 2 methods are too simple in use, to put usage example of them into doc string.